### PR TITLE
Cumulus 3904

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-3904](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3904)

Pulling in user contrib from #3801, thanks @mikedorfman  

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: tf-modules/cumulus/variables.tf [hunk 1]
```diff
@@ -626,3 +626,24 @@ variable "report_sns_topic_subscriber_arns" {
   default = null
   description = "Account ARNs to supply to report SNS topics policy with subscribe action"
 }
+
+variable "sqs_message_consumer_watcher_message_limit" {
+  type = number
+  default = 500
+  description = <<EOF
```
[the remark this answers is not in the document]
Comment (Jkovarik) [reply tf-modules/cumulus/variables.tf:633]: See: https://developer.hashicorp.com/terraform/language/expressions/strings#heredoc-strings

[the remark this answers is not in the document]
Comment (Jkovarik) [reply tf-modules/cumulus/variables.tf:633]: Super useful instead of super long strings for presentation, if nothing else.

**Review by jennyhliu — COMMENTED** [review]
Can we add to the example deployment? We can hardcode the values, just give user and example and help us to verify.
https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/main.tf#L221

**Comment by Jkovarik** [comment]
@jennyhliu  - while we *can* add anything to the Core module configuration in example, I'm not sure that we should use that as user documentation - the `/example` space should probably be named `core-devel` to match it's primary usage (integration tests).      I'd argue the primary source of Core module configuration should be either the variables file itself, or if we need a more elaborate demonstration, in the user webdocs *or* if applicable the template repo. 

If the configuration is *useful* to the integration tests/dev environment (e.g. as developers we want to enable devs to change this quickly for diagnosis/etc), that makes sense, but it feels YAGNI by default to me.

**Comment by jennyhliu** [comment]
> If the configuration is _useful_ to the integration tests/dev environment (e.g. as developers we want to enable devs to change this quickly for diagnosis/etc), that makes sense, but it feels YAGNI by default to me.

If we added the configurations to the example, we probably wouldn't miss the change in the first place.

**Review by jennyhliu — APPROVED** [review]
I have tested by passing these variables to cumulus module

**Comment by Jkovarik** [comment]
@jennyhliu  thanks for testing - I'm going to bring this up in the arch meeting, we *do* need to make sure this doesn't happen again.

**Pull request merged**